packages/manager/modules/anthos/src/dashboard/general-information/general-information.routing.js

```diff
@@ -0,0 +1,20 @@
+export default /* @ngInject */ ($stateProvider) => {
+  const stateName = 'anthos.dashboard.general-information';
+  $stateProvider.state(stateName, {
+    url: '/general-information',
```

Comment:
You can leave this as empty as the user can access dashboard at `/:serviceName` instead of `/:serviceName/general-information`

Reply:
This will also help in cleaner page `tracking` as well

Reply:
Changed.